src/app/pages/send/ordinal-inscription/coinselect/select-inscription-coins.ts

```diff
@@ -65,11 +77,20 @@ export function selectInscriptionTransferCoins(
     const [nextUtxo, ...remainingUtxos] = utxos;
     if (nextUtxo) neededInputs.push(nextUtxo);
     utxos = remainingUtxos;
-    txSize = txSizer.calcTxSize({
-      input_script: 'p2tr',
-      input_count: neededInputs.length + 1,
-      p2tr_output_count: 1,
-    });
+
+    if (isInscriptionInputNativeSegwit) {
+      txSize = getSizeInfo({
+        inputLength: neededInputs.length + 1,
+        outputLength: 1,
+        recipient,
+      });
+    } else {
+      txSize = txSizer.calcTxSize({
+        input_script: 'p2tr',
+        input_count: neededInputs.length + 1,
+        p2tr_output_count: 1,
+      });
+    }
```

Comment:
Wonder if it's worth revisiting the api we expose from this fee calculator. Maybe a wrapper over it?

Reply:
removed this